src/AutoSplitImage.py

```diff
@@ -98,6 +128,18 @@ def __init__(self, path: str):
         else:
             self.image_type = ImageType.SPLIT
 
+    def __parse_text_file(self, path: str):
```

Comment:
The `shutil.which` comment made me realize. We could totally catch a lack of tesseract install when trying to parse a text split. Something along the lines of:
```suggestion
    def __parse_text_file(self, path: str):
      if not TESSERECT_PATH:
        error_messages.tesseract_missing()
        return
```
Where `TESSERECT_PATH` comes from utils (the result of `shutil.which("tesseract")`)

In `error_messages.py`
```py
def tesseract_missing(ocr_split_file_path: str):
    set_text_message(
        f"{ocr_split_file_path!r} is an Optical Character Recognition split file but tesseract couldn't be found."
        + f'\nPlease read <a href="https://github.com/{GITHUB_REPOSITORY}#install-tesseract">'
        + f"github.com/{GITHUB_REPOSITORY}#install-tesseract</a> for installation instructions.",
    )
```